tests/testcafe/tests.js

```diff
@@ -59,7 +88,12 @@ function publicLibraryTest(game, variant, md5, tests) {
     await t
       .pressKey('esc')
       .click('#statesButton')
-      .click(Selector('td.name').withExactText(game).prevSibling().child())
+    if(!await Selector('td.name').visible) {
+      await t.hover(`.roomState:nth-of-type(1)`)
+        .click(`.roomState:nth-of-type(1) .edit`)
+        .click('p > .remove');
+    }
+    await t.click(Selector('td.name').withExactText(game).prevSibling().child())
```

Comment:
This shouldn't be necessary. See comment above. The main job of TestCafé is automated testing inside GitHub. There it gets a clean environment every time it is run.

When working on the TestCafé tests, you might have to cleanup the room manually by going to `http://localhost:8272/testcafe-testing`.